server/src/main/scala/com/cloudera/livy/server/LivyServer.scala

```diff
@@ -135,6 +161,32 @@ class LivyServer extends Logging {
     sys.props("livy.server.serverUrl") = _serverUrl.get
   }
 
+  def runKinit(livyConf: LivyConf): Unit = {
+    val keytab = livyConf.get(LAUNCH_KERBEROS_KEYTAB)
+    val principal = livyConf.get(LAUNCH_KERBEROS_PRINCIPAL)
+    require(principal != null,
+      s"Kerberos requires ${LAUNCH_KERBEROS_KEYTAB.key} to be provided.")
+    require(keytab != null,
+      s"Kerberos requires ${LAUNCH_KERBEROS_PRINCIPAL.key} to be provided.")
+    val commands = Seq("kinit", "-kt", keytab, principal)
+    val proc = new ProcessBuilder(commands: _*).start()
+    proc.waitFor() match {
+      case 0 =>
+        info("Run kinit command successfully")
```

Comment:
s/Run/Ran

I also think this should be less than info... it doesn't really provide a lot of interesting info and will just add noise to the logs.